ipatests/test_xmlrpc/test_automount_plugin.py

```diff
@@ -368,7 +369,8 @@ def test_2_automountmap_add_duplicate(self):
         """
         Test adding a duplicate direct map.
         """
-        res = api.Command['automountmap_add_indirect'](self.locname, self.mapname, **self.direct_kw)['result']
+        api.Command['automountmap_add_indirect'](
```

Comment:
You may want to keep the test for 'result' in the output with:
assert 'result' in api.Command...